components/atom/textarea/src/index.scss

```diff
@@ -0,0 +1,29 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '~@s-ui/react-atom-help-text/lib/index';
+@import '~@s-ui/react-atom-label/lib/index';
+@import '~@s-ui/react-atom-validation-text/lib/index';
+
+$short-num-lines: 5 !default;
+$long-num-lines: 7 !default;
+
+$pm-atom-textarea: $p-m !default;
+$fz-atom-textarea: $fz-base !default;
+$lh-atom-textarea: $lh-l !default;
+
+.sui-AtomTextarea {
+  font-size: $fz-atom-textarea;
+  padding-left: $p-l;
+  padding-top: $pm-atom-textarea;
+  line-height: $lh-atom-textarea;
+
+  &[disabled] {
```

Comment:
Usually I use only class selectors for performance concern. But if you're sure it's not a problem here it's fine to me.